Gemfile

```diff
@@ -10,20 +10,21 @@ end
 ruby '2.4.1'
 gem 'rails', '~> 5.1.5'
 
-gem 'acts_as_paranoid', github: 'ActsAsParanoid/acts_as_paranoid' # Need to point to repo or can't migrate: https://github.com/ActsAsParanoid/acts_as_paranoid/issues/36
+# https://github.com/ActsAsParanoid/acts_as_paranoid/issues/36
+gem 'acts_as_paranoid', github: 'ActsAsParanoid/acts_as_paranoid'
 gem 'bcrypt' # Use ActiveModel has_secure_password
 gem 'bootstrap'
 gem 'font-awesome-rails' # https://github.com/bokmann/font-awesome-rails
 gem 'haml-rails'
 gem 'honeybadger'
-gem 'jbuilder' # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
```

Comment:
I love how you are optimizing comments. And I thought just breaking the build on a Rubocop warning was a big step.

Reply:
Haha, yeah. I like to think of it as a "creative constraint".